solutions/client-landing-zone/setters.yaml

```diff
@@ -39,17 +39,23 @@ data:
   ##########################
   #
   # Name for the client, lowercase only
+  # customization: required
   client-name: 'client1'
-  # Billing Account ID to be associated with projects
+  #
+  # Clients Billing Account ID to associate with the client-landing-zone resources. The client-name-projects-sa will need billing user role permission granted.
```

Comment:
optional but a comment on "BAU or BAA role in the billing side - not the IAM side"
There are two sets of roles for billing and iam